### PR TITLE
REMOVE EVERYTHING

### PR DESCRIPTION
**Pull request by mathyoudawson**

### CODE REVIEW

File: lib/climate/climate.rb [hunk 1]
```diff
@@ -1,30 +0,0 @@
-# works out how hot the day is based on the last one
-# require_relative "climate_view.rb"
-require 'byebug'
-
-
-class Climate​
-  attr_accessor :temperature, :random_generator # , :view
-
-  VIEW = ClimateView.new
```
Comment (FunnyHector) [inline lib/climate/climate.rb:9]: You asked "can a constant be an object". Answer is yes. But should this be?

I think making the view as an instance variable would be better. Which you did in earlier versions but commented out now.

File: lib/climate/climate.rb [hunk 2]
```diff
@@ -1,30 +0,0 @@
...
-
-  def initialize
-    initial_temperature
-    @random_generator = Random.new
-    #@view = ClimateView.new
-  end
-
-  def initial_temperature
-    @temperature = rand(35)
-  end
```
Comment (FunnyHector) [inline lib/climate/climate.rb:19]: I would prefer to just put `@temperature = rand(35)` into the `#initialize` method. Why does the reader have to do one more method jump to read this line?

File: lib/climate/climate.rb (hunk 1 above)
Comment (FunnyHector) [reply lib/climate/climate.rb:9]: To add to this thread, a constant should be something constant, it shouldn't be changed by itself or other things. 

Just as the name, constant, suggests.

Comment (mathyoudawson) [reply lib/climate/climate.rb:9]: If i am only ever using the client view to output message and the object is self is never changes, only referenced, then is it not changing?

File: lib/climate/climate.rb (hunk 2 above)
Comment (mathyoudawson) [reply lib/climate/climate.rb:19]: Original idea was to make it more readable. i.e. making it obvious that I am setting the initial temp. But considering that this method is only one line I agree that it should be in initialize 💯

**Pull request closed without merging**